migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Looks like this and the other files in `migrations/` are duplicates of things already in the `alembic/` folder - can you consolidate?

Reply:
Do we still use `flask-migrate`, like in the `README`? That is the only reason I ended up generating the `migrations/` folder because I couldn't get the database upgrade procedure described there to work otherwise.

Reply:
Some back of the envelope math: 
- ~90000 libraries
- on average 20% are edited once monthly (This comes from nowhere in particular, just a number)
we would expect this table to be comparable to or even double the size of the original library table in roughly a year. I'd estimate the annual growth would be roughly in that same ballpark every year after.